### PR TITLE
Skipping CR cleanup validation in case of GKE

### PR DESCRIPTION
**Pull request by atrivedi-px**
**What this PR does / why we need it**:
Skip CR cleanup validation in case of GKE

**Which issue(s) this PR fixes** (optional)
Closes #PB-5226

**Special notes for your reviewer**:
Jenkins Run: https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/basic-system-test/job/px/job/s3/job/gke-cloud-px-backup-stork-upgrade/22/execution/node/267/log/?consoleFull

### CODE REVIEW

**Review by sn-px — APPROVED** [review]
LGTM

File: tests/backup/backup_helper.go [hunk 1]
```diff
@@ -6364,6 +6364,14 @@ func dumpMongodbCollectionOnConsole(kubeConfigFile string, collectionName string
 // validateCRCleanup validates CR cleanup created during backup or restore
 func validateCRCleanup(resourceInterface interface{},
 	ctx context.Context) error {
+
+	log.InfoD("Validating CR cleanup")
+
+	if GetClusterProviders()[0] == "gke" {
+		log.Infof("Skipping CR cleanup validation in case of GKE")
+		return nil
+	}
```
Comment (mkoppal-px) [inline tests/backup/backup_helper.go:6373]: Add a `TODO` to remove this code once we figure out how get get the correct kubeconfig in case of GKE

Comment (atrivedi-px) [reply tests/backup/backup_helper.go:6374]: Done

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**